css/index.css

```diff
@@ -143,4 +218,50 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+/* ===================About Page=================== */
+
+.top-content .text-container-about {
+    width: 96%;
+    padding: 0 1%;
+    padding-bottom: 20px;  
+}
+
+.about-middle-content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
```

Comment:
If you don't need this CSS just remove it before your final commit on a project.